nav2_theta_star_planner/README.md

```diff
@@ -51,8 +51,7 @@ The parameters of the planner are :
 - ` .how_many_corners ` : to choose between 4-connected and 8-connected graph expansions, the accepted values are 4 and 8
 - ` .w_euc_cost ` : weight applied on the length of the path. 
 - ` .w_traversal_cost ` : it tunes how harshly the nodes of high cost are penalised. From the above g(neigh) equation you can see that the cost-aware component of the cost function forms a parabolic curve, thus this parameter would, on increasing its value, make that curve steeper allowing for a greater differentiation (as the delta of costs would increase, when the graph becomes steep) among the nodes of different costs.
-- ` .w_heuristic_cost ` : it has been provided to have an admissible heuristic
-
+- `w_heuristic_cost ` : it has been provided to have an admissible heuristic; it is **not** a user configurable parameter
```

Comment:
Should this be removed here if not a parameter?

Reply:
Update needs to be made to remove it from navigation.ros.org as well

Reply:
I just put it in, cause I do use it in my explanation of the cost function

Reply:
I think you can just explain it there instead (that there's this thing around that is set as 1.0 or set to the same value, if less than 1.0)

Reply:
Ok so I should be changing the comment to state that it is set as min(1.0, w_euc_cost_), right?

Reply:
yeah

Reply:
@Anshu-man567 can you do this soon? This is actually blocking the galactic sync in https://github.com/ros-planning/navigation2/pull/2561